8. Application/superknowa-app/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Use the official Node.js v14 image as the base image
+FROM node:19
+
+# Set the working directory to /app
+WORKDIR /app
+
+# Copy the package.json and package-lock.json files to the container
+COPY package*.json ./
+
+# Install the dependencies
+RUN npm install
+
+# Copy the rest of the application files to the container
+COPY ./src ./src
+COPY ./public ./public
+COPY ./.env ./.env
```

Comment:
@hitalukder avoid adding .env if it has credentials in it.

Reply:
Good point. Removed the  line .env copy